_pages/proton-c.md

```diff
@@ -26,10 +26,27 @@ You can find the Proton C for purchase at:
 
 Support for Split Keyboards, RGB Underglow, and Backlighting are being worked on and should be available soon.
 
+## Comparison
+
+If you want to know why you should chose the Proton C for your next product, here is a comparison of the chips:
+
+---- | ATMega32u4        | AT90USB1286 | STM32F303xC
```

Comment:
Do you even visual alignment, bro?

```markdown
 &nbsp;     | ATMega32u4       | AT90USB1286          | STM32F303xC
----------- | ---------------- | -------------------- | ----------------
Speed       | 16MHz            | 16MHz                | 72MHz
Voltage     | 5v               | 5v                   | 3.3v with some 5v capable pins
Flash size  | 32KB (28 usable) | 128KB (120KB usable) | 256KB (248KB usable)
EEPROM size | 1024 bytes       | 4096 Bytes           | 4096 bytes (emulated in flash)
Memory      | 2.5KB            | 8KB                  | 48KB
Timers      | 4 total          | 4 total              | 10 total
Digital IO  | 25 pins          | 46 pins              | 37 pins
```

Reply:
Nope, didn't bother.  GitHub will render it just fine. :D